### DISCUSSION TITLE
How to configure dev server port

### DISCUSSION THREAD

**Discussion by zizhengtai** [wxt-dev/wxt#576] [Q&A] [answered]
I haven't found a way to tell WXT to not use the default 3000 port. (I know WXT tries to look for the next few available ports if 3000 isn't available, but I need to use a specific one.) Setting `vite.server.port` in `wxt.config.ts` has no effect.

**Comment by aklinker1** [comment]
There is no config for this right now. It'd be pretty easy to add, if anyone wants to contribute it.

https://github.com/wxt-dev/wxt/blob/main/src%2Fcore%2Fcreate-server.ts#L43

**Reply by zizhengtai** [reply]
I opened https://github.com/wxt-dev/wxt/pull/577. Let me know if I'm missing anything.

**Reply by aklinker1** [reply, marked as answer]
Thanks for the PR. Merged and released in [v0.17.10](https://github.com/wxt-dev/wxt/releases/tag/v0.17.10)!

```sh
wxt --port 1234
```

Or 

```ts
// wxt.config.ts
export default defineConfig({
  dev: {
    server: {
      port: 1234,
    },
  },
});
